docs/getting-started/running-release.md

```diff
@@ -43,66 +44,64 @@ $ tree
 └── target
     └── x86_64-unknown-linux-musl
         └── release
-            ├── bpfctl
-            └── bpfd
+            ├── bpfman
+            └── bpfman-rpc
 ```
 
-To deploy `bpfd`:
+To deploy `bpfman`:
 
 ```console
-sudo RUST_LOG=info ./target/x86_64-unknown-linux-musl/release/bpfd
-[2023-10-13T15:53:25Z INFO  bpfd] Log using env_logger
-[2023-10-13T15:53:25Z INFO  bpfd] Has CAP_BPF: true
-[2023-10-13T15:53:25Z INFO  bpfd] Has CAP_SYS_ADMIN: true
+sudo RUST_LOG=info ./target/x86_64-unknown-linux-musl/release/bpfman-rpc --timeout=0
+[INFO  bpfman::utils] Log using env_logger
```

Comment:
I feel like having the logs here(and everywhere) is just a huge maintenence burden, i.e in most places I think just posting the command *should* be good enough